examples/jsm/renderers/common/Pipelines.js

```diff
@@ -205,7 +205,7 @@ class Pipelines extends DataMap {
 
 		}
 
-		super.delete( object );
+		return super.delete( object );
```

Comment:
This method is overriding `DataMap.delete` which removes the deleted object. For consistency with how the base method works, it seems best to continue returning the deleted object.